adcm_aio_client/core/objects/_base.py

```diff
@@ -84,3 +90,24 @@ def __init__(self: Self, parent: Parent, requester: Requester, data: dict[str, A
 
     def get_own_path(self: Self) -> Endpoint:
         return *self._parent.get_own_path(), self.PATH_PREFIX, self.id
+
+
+class MaintenanceMode:
+    def __init__(
+        self: Self, maintenance_mode_status: MaintenanceModeStatus, requester: Requester, path: Endpoint
+    ) -> None:
+        self.maintenance_mode_status = maintenance_mode_status
+        self._requester = requester
+        self._path = path
+
+    def __repr__(self: Self) -> MaintenanceModeStatus:
+        return self.maintenance_mode_status
+
+    def __str__(self: Self) -> MaintenanceModeStatus:
+        return self.maintenance_mode_status
+
+    async def on(self: Self) -> None:
+        await self._requester.post(*self._path, "maintenance-mode", data={"maintenanceMode": MaintenanceModeStatus.ON})
```

Comment:
update internal value

note that it may become `changing` instead of `on`

Reply:
done

Reply:
nope, see about `changing`

Reply:
changed